src/Plugin/Visualisation/Style/AxisChart.php

```diff
@@ -853,6 +853,9 @@ protected function chartBuildSettings(array $records) {
       }
     }
 
+    // Get the row headers
+    $ticks_custom = $settings['axis']['x']['tick']['values']['custom'] ?? [];
```

Comment:
@WilsonWu2100

Reply:
@jez500 @jez500  This is because the original value of $settings['axis']['x']['tick']['values']['custom'] has been overwritten on line 886, that's why I add this line a bit earlier.